Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.cpp

```diff
@@ -319,7 +319,7 @@ URL ApplicationManifestParser::parseScope(const JSON::Object& manifest, const UR
 
 Color ApplicationManifestParser::parseColor(const JSON::Object& manifest, const String& propertyName)
 {
-    return CSSParser::parseColorWithoutContext(parseGenericString(manifest, propertyName));
+    return (Color)CSSParser::parseColorWithoutContext(parseGenericString(manifest, propertyName));
```

Comment:
C-style casts like these are not kosher in the WebKit coding style. If it needs to be a cast, it should be a static_cast, but I think replacing the `operator Color` on StyleColor with a function would be more idiomatic and clearer.

Reply:
Yes I will remove the conversion operator and use a regular well named function.